packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -33,18 +34,39 @@ export const MenuList = compose<MenuListType>({
   ...stylingSettings,
   slots: {
     root: MenuStack,
+    focusZone: FocusZone,
```

Comment:
Do we need a platform check here?

Reply:
I haven't done a Platform check in this particular place in the past, so I elected to not. I could try doing so and see if it makes any difference.

Reply:
I think my take on this is I'd rather not have slots only be defined on one platform. I'd rather have slots be defined on all platforms (and have their slotProps calculated) for the sake of consistency. I gave similar feedback in the PR #2103 about the new android only slot. So I'll leave it as is.